docs/designers-developers/developers/tutorials/javascript/extending-the-block-editor.md

```diff
@@ -0,0 +1,67 @@
+
+## Extending the Block Editor
```

Comment:
This should be an h1, and the opening blank newline should be removed.

Reply:
Updated here and for each sub page in 6672b8340